src/app/(content-pages)/transform-property/TransformPropertyPage.tsx

```diff
@@ -16,11 +17,15 @@ import park from "@/images/install-a-park.jpeg";
 import ContentCard from "../../../components/ContentCard";
 
 export default function TransformPropertyPage() {
+  const parkInATruckUrl =
+    "https://www.jefferson.edu/academics/colleges-schools-institutes/architecture-and-the-built-environment/programs/landscape-architecture/park-in-a-truck.html";
+  // update below url
+  const jumpStartUrl = "#";
```

Comment:
after some googling i found this link here, I think we can use it as a placeholder for now until @thansidwell or @nlebovits confirm what other links we should use: https://www.phila.gov/services/payments-assistance-taxes/taxes/tax-credits/business-tax-credits/jump-start-philly/